### PR TITLE
Added two py files for pytest

### PR DESCRIPTION
**Pull request by ivanbarcelos90**
# Description

Added py file for testing pytest in a basic math function.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

It was tested using pytest in pycharm IDE. 
The math function "pow" was used as an example for testing the passed and failed status of pytest.

- [x] Test A : 2 by the power of 2 must equal 4. Test passed. 
- [x] Test B: -2 by the power of 2 must equal 4. Test passed.
- [x] Test C : 3 by the power of 3 must equal 9. Test failed.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by ivanbarcelos90** [comment]
Igor, you can review my first try at pull request.

File: finance_data_lake/app.py [hunk 1]
```diff
@@ -1,2 +1,6 @@
-def pow(x: int) -> int:
-    pass
\ No newline at end of file
+
+
+def power(x, y):
+    return pow(x, y)
+
+
```
Comment (ivtavares) [inline finance_data_lake/app.py:4]: Remove the empty lines

File: tests/test_app.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import app
+
+
+def test_app_passed():
+    assert app.power(2, 2) == 4
+    assert app.power(-2, 2) == 4
+
+
+def test_app_failure():
+    assert app.power(3, 3) == 9
```
Comment (ivtavares) [inline tests/test_app.py]: Add an empty line here

File: tests/test_app.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import app
+
+
```
Comment (ivtavares) [inline tests/test_app.py:3]: Please check pytest parametize

**Review by ivtavares — CHANGES_REQUESTED** [review]

File: finance_data_lake/app.py (hunk 1 above)
Comment (ivtavares) [reply finance_data_lake/app.py:4]: When you finish your modification, you should click on Resolve conversation

File: tests/test_app.py (hunk 2 above)
Comment (ivtavares) [reply tests/test_app.py]: When you finish your modification, you should click on Resolve conversation

**Comment by ivtavares** [comment]
We need to improve the Commit messages. "Pull request fix" does not say anything. I will search online for good patterns and send them to you

**Review by ivtavares — APPROVED** [review]

**Comment by ivtavares** [comment]
Anyway, it is approved. You can merge

**Pull request merged**